lambdas/TranscriptionJobStartLambdaFunction/index.js

```diff
@@ -26,8 +26,15 @@ const transcribe = new AWS.TranscribeService({ apiVersion: '2017-10-26' });
 exports.handler = async (event) => {
   console.log(JSON.stringify({ msg: 'State input', input: event }));
 
+  const extension = event.Artifact.Descriptor.Extension;
```

Comment:
I'd consider renaming this variable to `mediaFormat`, since we're not actually messing with the file extension of the artifact, just the format of the job that getting sent to Transcribe.

Reply:
fair.

Reply:
updatd